src/main.pl

```diff
@@ -0,0 +1,15 @@
+use 5.36.0;
+use strict;
+use warnings;
+use utf8;
+
+use Encode qw/decode_utf8/;
+use Data::Dumper;
+
+use lib '.';
+
+require 'lib/KamakuraTrash.pm';
+require 'lib/TrashInfoDeliver.pm';
```

Comment:
KamakuraTrash, TrashInfoDeliver はそれぞれpackage宣言がしてあるので、use lib で `lib/` を含むようにして、use でいけます。
```suggestion
use FindBin;
use lib "$FindBin::Bin/lib";
use KamakuraTrash;
use TrashInfoDeliver;
```

FindBinを使うと、実行ファイルの場所からの相対指定が楽にできます。

ちなみに、使おうとしているモジュールがPerlのコアに入っているのか、別途インストールが必要かは `corelist FindBin` として、corelistコマンドを実行すると分かります。

```console
$ corelist FindBin         

Data for 2022-05-27
FindBin was first released with perl 5.00307
$ corelist Module::CoreList 

Data for 2022-05-27
Module::CoreList was first released with perl v5.8.9
```